packages/common/container-definitions/src/runtime.ts

```diff
@@ -123,7 +123,9 @@ export interface IBatchMessage {
  * @alpha
  */
 export interface IContainerContext {
-	readonly options: ILoaderOptions;
+	// Used to be ILoaderOptions, this is staging for eventual removal.
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	readonly options: Record<string | number, any>;
```

Comment:
could this be a separate PR that decouples context from loader options. that would be non-breaking

Reply:
that could include container runtime, and data store runtime too, bascially decouple the lower layer first